### PR TITLE
Add myself

### PR DESCRIPTION
**Pull request by yqin43**
add_yuehan

### CODE REVIEW

File: source/people/yuehan.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Yuehan Qin
+================
```
Comment (bam241) [inline source/people/yuehan.rst]: underlining should be as long as the underlined text.

File: source/people/yuehan.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Contact Information
+--------------------
```
Comment (bam241) [inline source/people/yuehan.rst]:
```suggestion
-------------------
```

File: source/people/yuehan.rst [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+`yqin43@wisc.edu <mailto:yqin43@wisc.edu>`_
+
+Yuehan is an undergraduate assistant in CNERG. She is hoping to help out however she can!
+
+Research Interests
+-------------------
```
Comment (bam241) [inline source/people/yuehan.rst]:
```suggestion
------------------
```

File: source/people/yuehan.rst [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* Software Development
+* More TBD!
+Education
+----------
```
Comment (bam241) [inline source/people/yuehan.rst]:
```suggestion
---------
```

**Comment by yqin43** [comment]
Dear Baptiste,

Thank you very much! Now I have updated the main.rst file and pushed it to the cnerg repo.

Best,
Yuehan

**Comment by gonuke** [comment]
Looking forward to getting you added to the website @yqin43.

This PR contains an empty file named `”` and I don’t see the modified version of `main.rst`.  These are common hiccups for a first PR and getting some in person guidance can often help get it right and also give you a deeper understanding

**Comment by gonuke** [comment]
I found the modified `main.rst`  - is in the `add_myself` branch on `cnerg/cnerg.github.com` while this PR is in the branch `add_myself` on `yqin43/cnerg.github.com`

**Comment by gonuke** [comment]
Also, there have now been other changes to `main.rst`, so you might want to get some tips on how to "rebase" your branch from @kkiesling

File: source/people/main.rst [hunk 5]
```diff
@@ -107,6 +107,11 @@ Undergraduate Students & Interns
 |                                    |                                    |                                     |
 | :doc:`Piper Lincoln <plincoln>`    | :doc:`Matthew Nyberg <mnyberg>`    | :doc:`Brad Laufenberg <laufenberg>` |
 +------------------------------------+------------------------------------+-------------------------------------+
++------------------------------------+------------------------------------+-------------------------------------+
+|                                    |                                    |                                     |
+|                                    |                                    |                                     |
+| :doc:`Yuehan Qin <yuehan>`         |                                    |                                     |
++------------------------------------+------------------------------------+-------------------------------------+
```
Comment (kkiesling) [inline source/people/main.rst]: Rather than have a new row, you can add yourself as a fourth entry on the same row as the others.

**Comment by yqin43** [comment]
@gonuke @kkiesling @bam241 Thank you! I have made another edition to the main file now. Please have a look. Thank you very much!

**Comment by gonuke** [comment]
The last commit didn't quite achieve what you were after.  Instead of adding it to the previous row, it made is an error so that it can't be processed.

If you want to add it to the previous row of the table, you'll need to extend each line that makes that table.

**Comment by gonuke** [comment]
This looks good to me (LGTM) - thanks @yqin43

**Comment by yqin43** [comment]
Thank you!

**Pull request merged**